fontconfig/build.rs

```diff
@@ -1,5 +1,6 @@
 fn main() {
-    let dlopen = std::env::var("RUST_FONTCONFIG_DLOPEN").is_ok();
+    println!("cargo:rerun-if-env-changed=RUST_FONTCONFIG_DLOPEN");
```

Comment:
Ah, thanks for this improvement. I was having to do a lot of `cargo clean` calls when testing #12 to rerun build.rs (not so bad with sccache, but still a waste of time).